ci/deploy_app.yaml

```diff
@@ -26,6 +25,12 @@ params:
   MIN_REPLICAS:
   MAX_REPLICAS:
   TARGET_CPU_UTILIZATION_PERCENTAGE:
+  CDN_URL:
+  WEB_SERVER_TYPE:
+  WEB_SERVER_WORKERS:
+  WEB_SERVER_THREADS:
+  WEB_SERVER_UWSGI_ASYNC_CORES:
+  DATASTORE_USE_GRPC:
```

Comment:
Added these as part of this PR as they were previously missing. While it doesn't impact automated pipelines it does mean they can now be overridden by a local developer fly exec to a GCP env which is a useful approach.